specs/windows/etw_process_events.table

```diff
@@ -0,0 +1,29 @@
+table_name("etw_process_events")
+description("Windows process execution events.")
+schema([
+    Column("type", TEXT, "Event Type"),
+    Column("pid", BIGINT, "Process ID"),
+    Column("ppid", BIGINT, "Parent Process ID"),
+    Column("session_id", INTEGER, "Session ID"),
+    Column("flags", INTEGER, "Process Flags"),
+    Column("exit_code", INTEGER, "Exit Code - Present only on ProcessStop events"),
+    Column("path", TEXT, "Path of executed binary"),
+    Column("cmdline", TEXT, "Command Line"),
+    Column("username", TEXT, "User rights - primary token username"),
+    Column("token_elevation_type", TEXT, "Primary token elevation type - Present only on ProcessStart events"),
+    Column("token_elevation_status", INTEGER, "Primary token elevation status - Present only on ProcessStart events"),
+    Column("mandatory_label", TEXT, "Primary token mandatory label sid - Present only on ProcessStart events"),
+    Column("datetime", TEXT, "Event timestamp in log UTC format"),
+    Column("time_windows", BIGINT, "Event timestamp in Windows format", hidden=True),
+    Column("time", BIGINT, "Event timestamp in Unix format", hidden=True),
+    Column("eid", INTEGER, "Event ID", hidden=True),
+    Column("header_pid", BIGINT, "Process ID of the process reporting the event", hidden=True),
+    Column("process_sequence_number", BIGINT, "Process Sequence Number - Present only on ProcessStart events", hidden=True),
+    Column("parent_process_sequence_number", BIGINT, "Parent Process Sequence Number - Present only on ProcessStart events", hidden=True),
```

Comment:
Are these columns hidden because they are mostly useful for dev/debuggign?

Reply:
Is the process_sequence_number unique at some level? Would be useful to have a non-PID unique identifier for a process

Reply:
A corner case scenario on Windows called "PID reuse" can happen when the same PIDs get reused during OS runtime lifecycle. This is something very unlikely that OS developers decided to address by including the concept of the "process sequence number" on the ETW fields. I'm including this data as a hidden value and exposing this information in the unlikely event this is needed.

@zwass yes, these could be useful for debugging scenarios

@defensivedepth yes, they are unique

Reply:
@marcosd4h How unique? Sysmon generates a GUID for each new process creation (https://learn.microsoft.com/en-us/sysinternals/downloads/sysmon#event-id-1-process-creation) specifically because of this issues around PIDs. 

I would recommend that we do not hide the sequence number columns. For security use cases / correlation, some type of GUID would be better than PIDs.

Reply:
@marcosd4h Wanted to make sure my reply didn't get lost in all the other comments

Reply:
Hey @defensivedepth, sorry for the late response! I totally miss your response.

This sequence value the code is exposing is generated by the kernel itself. The code is just grabbing it from the ETW event and exposing it to the table. In other words, the code does not control this value; it just relies on the kernel-generated value to be unique for the current os boot session.

Reply:
> @marcosd4h this is mostly nits and some requests for clarification.
> 
> This is tricky to read as such a huge PR, but it definitely helps that you documented many of the class members and methods. Thanks!

@zwass Thanks for reviewing the PR! I pushed some changes to address the comments you made